script.js

```diff
@@ -0,0 +1,95 @@
+
+
+const resetButton = document.querySelector('button[data-behavior="reset"]');
+const sizeButton = document.querySelector('button[data-behavior="size"]');
+const rainbowButton = document.querySelector('button[data-behavior="rainbow"]');
+const blackButton = document.querySelector('button[data-behavior="black"]');
+const gameContainer = document.querySelector('div[data-behavior="container"]');
+
+let isRainbowPen = false
+
+sizeButton.addEventListener('click', () => {
+  makeGrid()
+});
+
+rainbowButton.addEventListener('click', () => {
+  isRainbowPen = true
+});
+
+blackButton.addEventListener('click', () => {
+  isRainbowPen = false
+});
+
+window.onload = function() {
+  grid(16);
+}
+
+//Create Grid
+function grid(size) {
+  for(let i = 1; i <= (size * size); i++) {
+    let gridCell = document.createElement('div');
+    gridCell.classList.add('gridDivs');
+    gameContainer.appendChild(gridCell);
+    gridCell.addEventListener('mouseover', () => colorGridCell(gridCell));
+  }
+}
+
+//Resets the game and prompts the user for a numerical input for the size of the new grid.
+function makeGrid(gridSize) {
```

Comment:
This accepts an argument but is never passed one. Also this function doesn't seem to make a grid, the name is misleading